### ISSUE TITLE
syntax error when parsing date/time operators in postgres

### ISSUE THREAD

**Issue by uqichi** [k0kubun/sqldef#263]
The point at issue is usage of date/time operator like `- INTERVAL '1 YEAR'` or `- '1 year'::interval`

## Input SQL
```sql
CREATE MATERIALIZED VIEW IF NOT EXISTS points_view AS
    SELECT * FROM points
    WHERE id IN (SELECT MAX(id) AS id
        FROM points
        WHERE to_timestamp(concat(to_char(current_date - INTERVAL '1 YEAR' + INTERVAL '1 DAY', 'YYYY-MM-DD'), ' 00:00:00'), 'YYYY-MM-DD HH24:MI:SS') < created_at
        AND created_at <= to_timestamp(concat(to_char(current_date, 'YYYY-MM-DD'), ' 23:59:59'), 'YYYY-MM-DD HH24:MI:SS')
        GROUP BY main_type, sub_type, user_id)
```
## Current output
```sql
found syntax  when parsing DDL "CREATE MATERIALIZED VIEW IF NOT EXISTS points_view AS
    SELECT * FROM points
    WHERE id IN (SELECT MAX(id) AS id
        FROM points
        WHERE to_timestamp(concat(to_char(current_date - INTERVAL '1 YEAR' + INTERVAL '1 DAY', 'YYYY-MM-DD'), ' 00:00:00'), 'YYYY-MM-DD HH24:MI:SS') < created_at
        AND created_at <= to_timestamp(concat(to_char(current_date, 'YYYY-MM-DD'), ' 23:59:59'), 'YYYY-MM-DD HH24:MI:SS')
        GROUP BY main_type, sub_type, user_id)": syntax  at position 262 near 'interval'
```

## Expected output
No errors.

**Comment by k0kubun** [comment]
Fixed in v0.12.6.

Note that both https://github.com/k0kubun/sqldef/issues/262 and https://github.com/k0kubun/sqldef/issues/263 were invalid DDLs by themselves because you didn't write `CREATE TABLE points`. Next time, please consider testing your input on psql.

**Comment by uqichi** [comment]
Thank you so much. And I'm sorry for the incomplete DDLs missing a related table.
